src/components/WhaleSightingViewer/SightingList.scss

```diff
@@ -0,0 +1,65 @@
+@import "../../Constants.scss";
+
+.whale-sighting-posts{  
+    grid-row-start: 2;
+    grid-row-end: 3;
+    grid-column-start: 1;
+    grid-column-end: 4;
+
+    list-style: none;
+    display: flex;
+	justify-content: space-between;
+    flex-wrap: wrap;
+	
+    .whale-sighting-post{
+        margin-bottom: 5%;
+        border: solid 5px grey;
+
+        p {
+            color: $dark-font-color;
+            font-weight: $fontNormal;
+        }
+    }
+
+    @media screen and (max-width: 430px){
+        .whale-sighting-post{
+            width: 90%;
+        }
+    }
+
+    @media screen and (min-width: 431px){
+        .whale-sighting-post{
+            width: 45%;
+        }
+    } 
+
+    @media screen and (min-width: 1100px){
+        .whale-sighting-post{
+            width: 30%;
+        }
+    }
+}
+
+.post-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.sighting-info {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    margin-left: 1rem;
+  }
+  
+  .sighting-info p {
+    margin: 0;
+    text-align: left;
+  }
+  
+  .sighting-info button {
+    margin-left: auto;
+    margin-top: 1%;
+  }
```

Comment:
Standard practice is to end each file with exactly one blank line - see how GitHub has added a ⛔ symbol here?